packages/ingest/S3ProviderClient.js

```diff
@@ -12,6 +12,14 @@ class S3ProviderClient {
     this.bucket = bucket;
   }
 
+  async connect() {
```

Comment:
Hmm, I don't having to add these methods to providers where they have no contextual meaning, but I assume we have no choice given how our provider clients are used?

Reply:
Not really. I can pull those empty functions out into a mix-in if you'd prefer.

Reply:
That or we could hide them in a base provider client that we extend, but it's 6 one way and a half-dozen the other

Reply:
0cfb14418